File: apps/console/BasketballConsole/BasketballConsole/Models/Participation.cs
using NBA.Models.ValueObjects;

namespace NBA.Models
{
    public class Participation
    {
        public string Id { get; set; }

        public string GameId { get; set; }

        public int PlayerId { get; set; }

        public string PlayerName { get; set; }

        public string TeamName { get; set; }

        public List<GamePlay> Plays { get; set; } = [];
    }
}

File: WebSite with CosmosDB/client/nba-statistics/src/models/Participation.ts
import { GamePlay } from './GamePlay';

export class Participation {
    gameId: string;
    playerId: string;
    plays: GamePlay[];

    constructor(
        gameId: string,
        playerId: string,
        plays: GamePlay[]
    ) {
        this.gameId = gameId;
        this.playerId = playerId;
        this.plays = plays;
    }
}
